version_control/Codurance_September2020/css/pages/industry-public-sector.css

```
@@ -35,23 +31,25 @@
   padding-bottom: var(--space-7);
 }
 
+.partners__header {
+  max-width: 55ch;
```

I wonder if we can use here the freed max-w instead a set value
```suggestion
  max-width: var(--freed-max-width);
```